package/python-kiwi-spec-template

```diff
@@ -56,10 +56,11 @@ Packager:       Marcus Schaefer <marcus.schaefer@suse.com>
 Group:          %{pygroup}
 Source:         %{name}.tar.gz
 Source1:        %{name}-rpmlintrc
+Source2:        kiwi-%{version}-py3-none-any.whl
```

Comment:
We definitely should not be including wheels as sources. We should be constructing them in the spec file.

Reply:
done